src/sentry/sentry_metrics/indexer/ratelimiters.py

```diff
@@ -11,67 +11,65 @@
     RequestedQuota,
     Timestamp,
 )
-from sentry.sentry_metrics.configuration import UseCaseKey, get_ingest_config
+from sentry.sentry_metrics.configuration import RateLimiterNamespace, get_writes_ratelimiter_config
 from sentry.sentry_metrics.indexer.base import FetchType, FetchTypeExt, KeyCollection, KeyResult
 from sentry.utils import metrics
 
 OrgId = int
 
 
-def _build_quota_key(use_case_id: UseCaseKey, org_id: Optional[OrgId] = None) -> str:
-    use_case_str = {
-        UseCaseKey.PERFORMANCE: "performance",
-        UseCaseKey.RELEASE_HEALTH: "releasehealth",
-    }[use_case_id]
+def _build_quota_key(namespace: RateLimiterNamespace, org_id: Optional[OrgId] = None) -> str:
 
     if org_id is not None:
-        return f"metrics-indexer-{use_case_str}-org-{org_id}"
+        return f"metrics-indexer-{namespace.value}-org-{org_id}"
     else:
-        return f"metrics-indexer-{use_case_str}-global"
+        return f"metrics-indexer-{namespace.value}-global"
 
 
 @metrics.wraps("sentry_metrics.indexer.construct_quotas")
-def _construct_quotas(use_case_id: UseCaseKey) -> Sequence[Quota]:
+def _construct_quotas(namespace: RateLimiterNamespace) -> Sequence[Quota]:
     """
     Construct write limit's quotas based on current sentry options.
 
     This value can potentially cached globally as long as it is invalidated
     when sentry.options are.
     """
-    if use_case_id == UseCaseKey.PERFORMANCE:
+    if namespace == RateLimiterNamespace.PERFORMANCE:
         return [
-            Quota(prefix_override=_build_quota_key(UseCaseKey.PERFORMANCE, None), **args)
+            Quota(prefix_override=_build_quota_key(RateLimiterNamespace.PERFORMANCE, None), **args)
             for args in options.get("sentry-metrics.writes-limiter.limits.performance.global")
         ] + [
             Quota(prefix_override=None, **args)
             for args in options.get("sentry-metrics.writes-limiter.limits.performance.per-org")
```

Comment:
I'm not sure about the best way to deal with the options here since those are already registered

Reply:
I think the easiest way would be to not touch the quota logic with this change. So leave this as UseCaseKey and use the same quota depending on the use case regardless of whether the backend is cloudspanner or postgres.

Reply:
but where do you get the `UseCaseKey` from then ? are you suggesting hard coding in something like: 


```python

if namespace in [RateLimiterNamespace.PERFORMANCE_PG, RateLimiterNamespace.PERFORMACE_CS]:
    # set quotas using UseCaseKey.Performance....
elif namespace in [RateLimiterNamespace.RELEASE_HEALTH_PG, RateLimiterNamespace.RELEASE_HEALTH_CS]:
    # set quotas using UseCaseKey.Release Health...
```

?

Reply:
Oh, I was just thinking we could keep passing it into this function like before.